west.yml

```diff
@@ -17,3 +19,9 @@ manifest:
     - name: zephyr-xenlib
       remote: xentroops
       revision: wip_zephyr_dom0
+    - name: littlefs
+      remote: zephyr
+      path: modules/fs/littlefs
+      groups:
+        - fs
+      revision: ca583fd297ceb48bced3c2548600dc615d67af24
```

Comment:
This module is not in our repository. I don't think we have to make fork for this repository until we make changes here.

Reply:
why can't you put name of branch instead of commit hash even without fork?

Reply:
so, is our functionality written in such way that further updating of the liitlefs and fixing bugs by maintainers can break our functionality?.. updating some functionality should not break backward compatibility and I am sure that maintainers of the littlefs knew about that, at least, we can stay on some release branch where everything is stable

Reply:
> updating some functionality should not break backward compatibility and I am sure that maintainers of the littlefs knew about that,

I'm not so sure about that. That's why this is a common way to use tags, not branches for the releases (linux kernel for example) and all references to the exact kernel is based on tag, not branch. Basing on branch, even release branch is bad practice because we can't be sure that your build = customer build. And what if some bug appeared in littlefs with newly pushed commits? Are you ready to spend time fixing it?

Reply:
so, do you able to use tag instead of hash here?

Reply:
> And what if some bug appeared in littlefs with newly pushed commits? Are you ready to spend time fixing it?

Actually if you stay on stable tag or released branch, I think you should not care about that

Reply:
I don't know if I'm able to use tag here. I haven't looked on this.

> Actually if you stay on stable tag or released branch, I think you should not care about that
this is the my point

I don't accept branch, even release branch based on branch and tag definition in GIT.

Reply:
This is not our repository. We can't set any tags or branches. I can't see any reason to fork this repo for chenging this module.

Reply:
@dsemenets  got it, thanks for the answer